urbandict-search/README.md

```diff
@@ -404,45 +384,19 @@ components:
   - !NumpyIndexer
     with:
       index_filename: vec.gz
-      metrix: cosine
-  - !ChunkPbIndexer
+      metric: cosine
+    metas:
+      name: vecidx  # a customized name
+      workspace: $TMP_WORKSPACE
+  - !BinaryPbIndexer
     with:
       index_filename: chunk.gz
-requests:
-  on:
-    IndexRequest:
-      - !VectorIndexDriver
-        with:
-          executor: NumpyIndexer
-      - !PruneDriver
-        with:
-          level: chunk
-          pruned:
-            - embedding
-            - buffer
-            - blob
-            - text
-      - !KVIndexDriver
-        with:
-          level: chunk
-          executor: BasePbIndexer
-    SearchRequest:
-      - !VectorSearchDriver
-        with:
-          executor: NumpyIndexer
-      - !PruneDriver
-        with:
-          level: chunk
-          pruned:
-            - embedding
-            - buffer
-            - blob
-            - text
-      - !KVSearchDriver
-        with:
-          level: chunk
-          executor: BasePbIndexer
-
+    metas:
+      name: chunkidx  # a customized name
+      workspace: $TMP_WORKSPACE
+metas:
+  name: chunk_indexer
+  workspace: $TMP_WORKSPACE
 ```
 
 As the same as the `doc_indexer`, the `chunk_indexer` also have different behaviors on different requests. For the `IndexRequest`, the `chunk_indexer` uses three Drivers in serial, namely, `VectorIndexDriver`, `PruneDriver`, and `KVIndexDriver`. The idea is to firstly use `VectorIndexDriver` to call the `index()` function from the `NumpyIndexer` so that the vectors for all the Chunks are indexed. Afterwards the `PruneDriver` prune the message, and the `KVIndexDriver` calls the `index()` function from the `BasePbIndexer`. This behavior is defined in the `executor` field.
```

Comment:
The readme is outdated, for example `PruneDriver` has been removed, gonna be misleading

Reply:
readmes seem very outdated. Some parts need big factor. For example some `request:` was removed previously but the readme wasn't updated. How about do it later after migrating?

Reply:
I agree @YueLiu-jina . Refactoring now all README.md will be too much

Reply:
fine with me :)